src/api/.rubocop.yml

```diff
@@ -115,6 +117,11 @@ RSpec/FilePath:
   Exclude:
     - 'spec/controllers/webui/package_controller/binaries.spec'
 
+# We disabled this cop because named subject isn't a better construction then
+# using let
+RSpec/NamedSubject:
+  Enabled: false
```

Comment:
If the cop is now disabled, its TODOs have to be removed: https://github.com/openSUSE/open-build-service/blob/bbdab04f9c286cdcfec0bad546419f654845e115/src/api/.rubocop_todo.yml#L873-L974